prow/cmd/checkconfig/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 The Kubernetes Authors.
+Copyright 2018 The Kubernetes Authors.
```

Comment:
nit: is git diff confused or was the file moved? if it was moved we should leave the original date AFAIK /shrug

Reply:
I think it thinks that I moved the `config/main.go` but in reality I just deleted it.